airflow/models/pydantic/dag_run.py

```diff
@@ -42,7 +39,6 @@ class DagRunPydantic(BaseModelPydantic):
     last_scheduling_decision: Optional[datetime]
     dag_hash: Optional[str]
     updated_at: datetime
-    consumed_dataset_events: List[DatasetEventPydantic]
```

Comment:
I could not find this one in `airflow/models/dagrun.py`, so I removed it. Again, I might miss something